aspnetcore/security/enforcing-ssl.md

```diff
@@ -361,7 +361,7 @@ The Windows Subsystem for Linux (WSL) generates a HTTPS self-signed cert. To con
 
 ## Troubleshoot certificate problems
 
-This section provides help when the ASP.NET Core HTTPS development certificate has been [installed and trusted](#trust), but you still have browser warnings that the certificate is not trusted.
+This section provides help when the ASP.NET Core HTTPS development certificate has been [installed and trusted](#trust), but you still have browser warnings that the certificate is not trusted. The ASP.NET Core HTTPS development certificate that is used by [Kestrel](xref:fundamentals/servers/kestrel).
```

Comment:
I'm looking on the phone ... Hard to read a bit ... Is it missing a verb?

Reply:
Drop "that"?

Reply:
Or enjoy your evening and look tomorrow :)